firefox-ios/Client/Frontend/Home/Wallpaper/WallpaperBackgroundView.swift

```diff
@@ -0,0 +1,66 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import UIKit
+import Common
+import Redux
+
+class WallpaperBackgroundView: UIView {
+    // MARK: - UI Elements
+    private lazy var pictureView: UIImageView = .build { imageView in
+        imageView.image = nil
+        imageView.contentMode = .scaleAspectFill
+        imageView.clipsToBounds = true
+    }
+
+    // MARK: - Variables
+    var wallpaper: Wallpaper? {
```

Comment:
We get nice functionality on wallpaper like providing a default image as well as fetching an image for landscape vs portrait. Are you proposing that it would be better to pass in these as separate items to the wallpaper view? We definitely could I guess i am not sure what the benefit as `wallpaper` is already a lightweight struct.

Reply:
By adding wallpaper, we are surfacing other properties and logic that the view doesn't need to know. So by providing a specific state for the view, we only pull in what's needed from wallpaper. 

Also, we can follow a consistent structure in our codebase which our views take in a state, which I'm happy to rethink or restructure. So for other views, `PocketStandardCell` populates its data by passing in `PocketStoryState` and `HomepageHeaderCell` takes in `HeaderState`.

The downside of this that I can see is that now the views are heavily tied to a state instead of specific properties if we decide to move from Redux.

However, now that I think about it, maybe this should be a team discussion to see what is preferred. Let me know what you think!

This also doesn't need to block this PR. So happy to get this merge in first and have that discussion later.

Reply:
https://mozilla.slack.com/archives/C05C9RET70F/p1732034555214689

Reply:
It seems like the outcome of that discussion is we actually do want to have the view have a reference to wallpaper state with just the information it needs instead of the the wallpaper object.